### PR TITLE
Jrobinson/custom slack updates

### PR DESCRIPTION
**Pull request by JoelRobinson-Tista**
Added a number of constants that can be added to any Jenkinsfile or any other pipeline job. 
* src/gov/va/appeals/devops/Caseflow.groovy
* src/gov/va/appeals/devops/Environments.groovy
* src/gov/va/appeals/devops/Jenkins.groovy

Moved logic from generic-ansible.groovy to customSlack.groovy. Hopefully should be useful on any other job as well. 

Testing:
* Tested on the `builds/job/preprod/job/certification` and `deploys/job/preprod/job/certification`
* Tested for regressions on `devops/job/aws_cert_report-all`

### CODE REVIEW

File: src/gov/va/appeals/devops/Caseflow.groovy [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package gov.va.appeals.devops
+
+class Caseflow {
+static final CASEFLOW_APPS =  ['certification', 'efolder', 'monitor']
+static final CASEFLOW_DB_APPS =  ['certification', 'efolder']
```
Comment (JoelRobinson-Tista) [inline src/gov/va/appeals/devops/Caseflow.groovy:5]: I don't really like this name. If anyone has any suggestions please let me know.

Comment (aroltsch) [reply src/gov/va/appeals/devops/Caseflow.groovy:5]: Let me think about it

Comment (aroltsch) [reply src/gov/va/appeals/devops/Caseflow.groovy:5]: Do we even use `CASEFLOW_APPS` constant in here https://github.com/department-of-veterans-affairs/appeals-deployment/pull/3262/files?

Comment (aroltsch) [reply src/gov/va/appeals/devops/Caseflow.groovy:5]: Oh I see where you use it.

Comment (aroltsch) [reply src/gov/va/appeals/devops/Caseflow.groovy:5]: how about `CASEFLOW_APPS_WITH_DB`?

**Review by aroltsch — APPROVED** [review]

**Pull request merged**